Makefile

```diff
@@ -18,8 +18,21 @@ conda:
 	rm ${SYMBIFLOW_ARCHIVE}
 	cd third_party/prjxray && $(MAKE) build -j`nproc`
 
+run-tests:
+	python3 exhaust.py --build_type generic --fail
+
+run-parameters-tests:
+	python3 exhaust.py --parameters parameters.json --toolchain vpr --build_type parameters
+
+run-multiple-samples-tests:
+	for run in {0..10}; do \
```

Comment:
can we define this number as parameter? Preferably sth we can overwrite by setting an ENV variable. The dafault can be 10

Reply:
Done, set this to 2 as default actually, as this CI should just test that the behavior is correct.

With another script we can make use of the multiple test runs to have a continues build that is triggered on every HEAD change.